src/main/java/com/jland/website/model/serializer/LocalDateSerializer.java

```diff
@@ -0,0 +1,21 @@
+package com.jland.website.model.serializer;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class LocalDateSerializer extends StdSerializer<LocalDate> {
```

Comment:
please shed light why we need it

Reply:
It's for serializing LocalDate/LocalTime in proper way